Userland/Utilities/lsdev.cpp

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
I probably need to add this new utility to the `Userland/Utilities/CMakeLists.txt` file?

Reply:
Done. I added the utility to the recommended set.